files/fluentd/run.sh

```diff
@@ -68,8 +67,31 @@ fi
 # Determine allowed total given the number of outputs we have.
 ALLOWED_DF_LIMIT=$(expr $DF_LIMIT \* $ALLOWED_PERCENT_OF_DISK / 100) || :
 
+TOTAL_LIMIT_SIZE_CONF=$(grep "total_limit_size" $FLUENT_CONF |  head -1 | awk '{print $2}' )
```

Comment:
This is unrealiable if there are multiple outputs configured.  You need to pass the value from the spec as an env var from here: https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/fluentd.go#L432

Reply:
How to access these variable in run.sh ?